src/core.js

```diff
@@ -309,6 +309,21 @@ $.extend( $.validator, {
 				this.element( element.parentNode );
 			}
 		},
+		onchange: function( element, event ) {
+
+			// If target element is a select, trigger focusout on change
+			if ( element.tagName.toLowerCase() === "select" ) {
+				var _this = element;
+
+				if ( !_this.form && _this.hasAttribute( "contenteditable" ) ) {
+					_this.form = $( _this ).closest( "form" )[ 0 ];
+				}
+
+				var validator = $.data( _this.form, "validator" ),
+					settings = validator.settings;
+				settings.onkeyup.call( validator, _this, event );
```

Comment:
> Are you referring to _this.hasAttribute( "contenteditable" )?

Yes


> I was mostly moving the same logic from the original event into this block so I didn't miss anything.

No need for it.
Just noticed that that block is redundant with the one in function `delegate`, so you can omit it.